students/luke/s11/music_filter.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+
+"""
+https://canvas.uw.edu/courses/1188584/assignments/4189634?module_item_id=8331790
+
+Your job, now, is to get artists and song names for for tracks with
+danceability scores over 0.8 and loudness scores below -5.0. In other words,
+quiet yet danceable tracks. Also, these tracks should be sorted in descending
+order by danceability so that the most danceable tracks are up top.
+"""
+
+import pandas as pd
+
+
+def make_list(dataframe):
+    return \
+        [f"{x.get('name')} by {x.get('artists')}"
```

Comment:
This is throwing a syntax error for me. Can you fix this?

Reply:
Can you elaborate?  It works for me.

edi:s11 luke$ python3 --version
Python 3.6.3
edi:s11 luke$ ./music_filter.py 
['Bad and Boujee (feat. Lil Uzi Vert) by Migos', 'Fake Love by Drake', 'HUMBLE. by Kendrick Lamar', 'Bank Account by 21 Savage', "You Don't Know Me - Radio Edit by Jax Jones"]